GoogleForCreators/web-stories-wp
# Dashboard: Remove Saved Templates Related Code

## Context

Last summer we had scaffolded out the 'saved templates' view in the dashboard ahead of getting data flowing. There's still questions out about what all a saved template is and how to handle them. The dashboard has changed since then and maintaining this code is not worth it since it's out of date and getting saved templates working is not a current priority. At this point, it'll be easier to just add in what we need when we need it rather than try to update all of this.

## Summary

Remove all references to saved templates (bookmark, create template from story, in progress story actions) in dashboard as well as 3 defunct experiments (`enableInProgressStoryActions` and `enableBookmarkActions`, `enableInProgressViews`). 

Now the only reason we need `@web-stories-wp/story-editor` is to create a story from a template (`getStoryPropsToSave`). 

## Relevant Technical Choices

Just removing relevant code. 

## To-do

None.

## User-facing changes

Just removing experiments.

## Testing Instructions

Just some regression testing here. Make sure that all actions expected are present in the dashboard for stories and templates, and when you go to `wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/saved-templates` (saved templates route) you just get the same empty page you would if you went anywhere without a defined route in the dashboard.




This PR can be tested by following these steps:

1.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8746
